src/js/components/help/help.js

```diff
@@ -65,7 +70,7 @@ var Help =  createReactClass({
     return {
       snackbar: AppStore.getSnackbar(),
       hasMultitenancy: AppStore.hasMultitenancy(),
-      isHosted: (window.location.hostname === "hosted.mender.io"),
```

Comment:
is this by intention?

Reply:
I noticed that as I'd just made the pr - fixed already!

Reply:
Excellent!